apps/nextjs-website/react-components/types/Stats/Stats.types.ts

```diff
@@ -0,0 +1,13 @@
+export interface StatsItemProps {
+  readonly title: string;
+  readonly description?: string;
+  readonly iconURL?: string;
+}
+
+export interface StatsProps {
+  readonly eyelet?: string;
+  readonly title: string;
+  readonly body?: string;
+  stats: ReadonlyArray<StatsItemProps>;
+  sectionID?: string | null;
+}
```

Comment:
```suggestion
export interface StatsProps extends Omit<SectionProps, 'theme'> {
  readonly eyelet?: string;
  readonly title: string;
  readonly body?: string;
  stats: ReadonlyArray<StatsItemProps>;
}
```
Prefer SectionProps over manually adding sectionID into the interface.